activity/inference/activity.json

```diff
@@ -26,6 +26,16 @@
         "name": "framework",
         "type": "string",
         "required": true
+    },
+    {
+        "name": "sigDefName",
+        "type": "string",
+        "required": false
```

Comment:
we should set the default value here:

{
  "name": "sigDefName",
  "type": "string",
  "required": false,
  "value": "whatever the default is"
}